Makefile

```diff
@@ -0,0 +1,49 @@
+TAG     := $(shell git rev-parse --short HEAD)
+SERVER  := fosskers/gtsite-service
+NGINX   := fosskers/gtsite-nginx
+TRANSIT := fosskers/gt-transit
+CHATTA  := fosskers/gt-chatta-demo
+
+clean:
+	cd geotrellis-site && make clean
+	cd geotrellis-chatta-demo && make clean
+	cd geotrellis-transit && ./sbt clean
+
+# One-time data fetches/ingests
+setup: chatta
```

Comment:
Thus far, this particular situation is a little uncommon. The closest example we have is RF.

With RF, there is a repository that contains all of the services required to drive the application, but all of the Terraform configuration is in another repository:

- https://github.com/azavea/raster-foundry
- https://github.com/azavea/raster-foundry-deployment

The high level approach is that all of the artifacts required to drive a deployment are created as part of the CI process for the main repository (container images, JARs, etc.):

- https://github.com/azavea/raster-foundry/blob/develop/Jenkinsfile#L10-L36

From there, the deployment repository is pulled down after tests past for certain branches, and a new deployment is kicked off:

- https://github.com/azavea/raster-foundry/blob/develop/Jenkinsfile#L38-L64

`GIT_COMMIT` is passed to `infra`, which takes care of threading the value of that variable as the container image tag provided to ECS:

- https://github.com/azavea/raster-foundry-deployment/blob/develop/scripts/infra#L68

The flexibility of using `GIT_COMMIT` comes into play when we need to deploy old versions of the application (to reproduce a bug), new versions (test a new feature in AWS), or just different environments (staging vs. production). The main requirement is that images already exist in the container image registries with references to `GIT_COMMIT`.